package.json

```
@@ -19,7 +19,7 @@
     "test:e2e:local": "export IS_TEST=true && protractor protractor.local.config.js",
     "test:e2e:saucelabs": "export IS_TEST=true && protractor protractor.saucelabs.config.js",
     "test:lint": "eslint lib/**/*.js test/**/*.js",
-    "test:flow": "flow; test $? -eq 0 -o $? -eq 2",
+    "test:flow": "flow; flow stop; test $? -eq 0 -o $? -eq 2",
```

@basti1302 That won't work as intended. The `test` command at the end checks the exit status of the previous command and now it would check the exit status of `flow stop` instead of `flow`.

```suggestion
    "test:flow": "flow; test $? -eq 0 -o $? -eq 2; flow stop",
```

---

🤦  you're right. I'll reverse those two commands.

---

Updated that to capture the status code, stop the server and then check the status code.